lib/components/Notification/items-list/index.tsx

```diff
@@ -20,6 +20,7 @@ const NotificationItemsList = ({
     onClose,
     onMarkAsRead,
 }: NotificationItemsListProps) => {
+    const [activeItemId, setActiveItemId] = useState("");
```

Comment:
using state for the list on Notifications in order to keep buttons open only on the latest active item where the buttons have been open last since on iOS, `onBlur` doesn't let us achieve it because it fires on the root `div` of the list instead of every notification. On other devices, `onBlur` fires on every notification.